.qlty.yml

```
@@ -0,0 +1,91 @@
+version: 1
+
+plugins:
+  - name: "biome"
+    enabled: true
+    config:
+      format: true
+      lint: true
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)